integration-tests/src/test/java/tachyon/client/table/RawColumnIntegrationTest.java

```diff
@@ -27,21 +35,26 @@
  */
 public class RawColumnIntegrationTest {
   private LocalTachyonCluster mLocalTachyonCluster = null;
-  private TachyonFileSystem mTfs = null;
+  private TachyonFS mTfs = null;
+
+  @Before
+  public final void before() throws Exception {
+    mLocalTachyonCluster = new LocalTachyonCluster(10000, 1000, Constants.GB);
+    mLocalTachyonCluster.start();
+    mTfs = mLocalTachyonCluster.getOldClient();
```

Comment:
Why are we switching from the new client to the old client. That seems backwards?